yt/go/yt/integration/auth_client_test.go

```diff
@@ -4,6 +4,8 @@ import (
 	"context"
 	"crypto/sha256"
 	"fmt"
+	"go.ytsaurus.tech/yt/go/yt/ythttp"
+	"os"
```

Comment:
It should be linters' job to say that, but these imports are not sorted properly. I suggest you configure your IDE for auto sorting (and we also need lint workflow here of course)